dm/config/task.go

```diff
@@ -184,7 +183,6 @@ type SyncerConfig struct {
 	MetaFile    string `yaml:"meta-file" toml:"meta-file" json:"meta-file"` // meta filename, used only when load SubConfig directly
 	WorkerCount int    `yaml:"worker-count" toml:"worker-count" json:"worker-count"`
 	Batch       int    `yaml:"batch" toml:"batch" json:"batch"`
-	MaxRetry    int    `yaml:"max-retry" toml:"max-retry" json:"max-retry"`
```

Comment:
Does it affect the process start and stop, because some users may have already set this parameter？

Reply:
some users may not realize the meaning of `MaxRetry` , only care about whether data lost or not , and some internal error like `invalid connection` also counted as once retry, it's not proper for user to control retry count, and  i think it's a bit conflict with optimizing retry part, force remove this config is worth